cli/exec.go

```diff
@@ -95,19 +96,26 @@ func (e *ExecCommand) runExec(args []string) error {
 		wg.Add(1)
 		go func() {
 			defer wg.Done()
-			io.Copy(os.Stdout, reader)
+			if !e.Terminal {
```

Comment:
I think we should add some test case for `exec` command  `-i` and `-it` flags

we can add the test case later, because this feature is needed very hurry

Reply:
sure thing.